app/routes/admin/groups/components/GroupMembersList.js

```diff
@@ -68,6 +71,7 @@ const GroupMembersList = ({
   const GroupLinkRender = (abakusGroup) =>
     true && (
       <Link to={`/admin/groups/${abakusGroup}/members?descendants=false`}>
+        {/* eslint-disable-next-line react/prop-types */}
```

Comment:
What is this warning, and how do we fix it?

Reply:
This looks like a bug in eslint. They closed the issue but even though we have the newest version it still does not look like it has been fixed.

Reply:
Not an issue since we moved location to window.location